datahub-web-react/src/app/onboarding/config/PoliciesOnboardingConfig.tsx

```diff
@@ -15,12 +15,13 @@ export const PoliciesOnboardingConfig: OnboardingStep[] = [
                     Welcome to DataHub <strong>Policies</strong>!
                 </p>
                 <p>
-                    If you need fine-grained access controls, <strong>Policies</strong> will do the trick.
+                    In most cases, <strong>Roles</strong> can be used to grant privileges to users on DataHub.
```

Comment:
This seems to imply that Roles are not usable in some cases, which isn't the case. What we're really trying to get across is that sometimes they're not the best fit.

Can we say something like "In most cases, **Roles** are the best option for granting privileges to DataHub users."

Reply:
Very much agree!